content/3_Silver/DFS.problems.json

```diff
@@ -112,6 +112,18 @@
         "kind": "internal"
       }
     },
+    {
+      "uniqueId": "usaco-1159",
+      "name": "Connecting Two Barns",
+      "url": "http://www.usaco.org/index.php?page=viewproblem2&cpid=1159",
+      "source": "Silver",
+      "difficulty": "Normal",
+      "isStarred": false,
+      "tags": ["Binary Search", "DFS", "DSU", "Two Pointers"],
+      "solutionMetadata": {
+        "kind": "internal"
```

Comment:
revert the solution metadata

Reply:
also I'd remove the binary search / DSU / DFS tags and keep "Connected Components" as binary search isn't even necessary for this problem

Reply:
as mentioned previously, "Two Pointers" should be "2P" for consistency